tf_ops/3d_interpolation/tf_interpolate.py

```diff
@@ -51,8 +51,8 @@ def _three_interpolate_grad(op, grad_out):
         now = time.time() 
         for _ in range(100):
             ret = sess.run(interpolated_points)
-        print time.time() - now
-        print ret.shape, ret.dtype
+        print(time.time() - now)
+        print(ret.shape, ret.dtype)
```

Comment:
This is not quite right.  When you are printing multiple values separated by a comma then you must `from __future__ import print_function` at the top of the import statements to get identical output in Python 2.

$ __python2 -c "print 'a', 'b' ; print('a', 'b')"__
```
a b
('a', 'b')
```